invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -384,9 +384,9 @@ export class CreatibutorsModal extends Component {
     const selectedSuggestion = selectedSuggestions[0].extra;
     this.setState(
       {
-        organizationIdentifiers: selectedSuggestion.identifiers.map(
-          (identifier) => identifier.identifier
-        ),
+        organizationIdentifiers: selectedSuggestion.identifiers
+          .map((identifier) => identifier.identifier)
+          .filter((identifier) => !identifier.toLowerCase().includes("grid")),
```

Comment:
We could filter out by `scheme` equals `grid` instead.
This would need to be done before the `map` operation.

Reply:
Updated, for some reason before I wasn't getting the scheme for grid, just the value, it's working fine now